build.py

```diff
@@ -698,7 +712,7 @@ def create_dockerfile_linux(ddir, dockerfile_name, argmap, backends, repoagents,
         dfile.write(df)
 
 
-def dockerfile_prepare_container_linux(argmap, backends):
+def dockerfile_prepare_container_linux(argmap, backends, enable_gpu):
```

Comment:
You don't need to add an extra argument, just get value from `FLAGS.enable_gpu` when needed like how it is done elsewhere

Reply:
This specific function is used in `compose.py` that's why I'm passing it in as a param